src/main/java/AndreeaG/driver/BrowserManager.java

```diff
@@ -0,0 +1,4 @@
+package AndreeaG.driver;
+
+public class BrowserManager {
```

Comment:
You are missing in this class the method that creates the driver. Please add it and use it in your methods. Thanks!